profiles/base/deployment.yaml

```diff
@@ -13,6 +14,8 @@ spec:
         - $(userid-header)
         - "-userid-prefix"
         - $(userid-prefix)
+        - "-workload-identity"
+        - $(gcp-sa)
```

Comment:
Longer term:

An alternative pattern that might be more convenient would be to mount the config map as a file in the volume and then pass the config path as a command line argument.

Then overlays can just add parameters to that config map but the deployment doesn't have to change. 

Right now we are relying on kustomize to do the substitution when kustomize build is called.

Lets not do that now though because it would require changes to the profiler code.